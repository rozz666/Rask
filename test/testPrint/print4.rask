main() -> void
{
    print(1, 2);
}

