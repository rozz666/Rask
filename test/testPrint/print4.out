print4.rask:3:4: error: function print(int, int) not found

